cluster.go

```diff
@@ -223,7 +218,11 @@ func (c *clusterClient) _refresh() (err error) {
 	groups := result.parse(c.opt.TLSConfig != nil)
 	conns := make(map[string]connrole, len(groups))
 	for master, g := range groups {
-		conns[master] = connrole{conn: c.connFn(master, c.opt), replica: false}
+		opt := *c.opt
+		if c.opt.SendToReplicas != nil {
+			opt.ReplicaOnly = false
+		}
+		conns[master] = connrole{conn: c.connFn(master, &opt), replica: false}
```

Comment:
The current code above will allocate a copy of `c.opt` on the heap every time.
We should do it better by preparing the copied `opt` once in advance.

Reply:
[cd6a674](https://github.com/redis/rueidis/pull/430/commits/cd6a674ec54ce392de2ff6e0d268fc6cb877e344)

yes. you're right.